packages/engine-server/src/seed/service.ts

```diff
@@ -228,6 +231,7 @@ export class SeedService {
   }): Promise<SeedSvcResp> {
     const ws = new WorkspaceService({ wsRoot: this.wsRoot });
     const config = ws.config;
+    ws.dispose();
```

Comment:
instead of disposing of the workspace everytime, can we add a singleton by using [getOrCreate](https://github.com/dendronhq/dendron/blob/c6cd54f99dc9c17e28566bbe80f3c23546fe935d/packages/lsp-server/src/engine-server.ts#L15)?

Reply:
Do you mean a global singleton for the `WorkspaceService`? I'm reluctant to implement it that way because some code could create one with a different `wsRoot`. We could go with a memoized `getOrCreate`, but we'll still be opening files and leaving them open as we keep workspace services in the memoize cache.

I simplified some of the code by using a static function instead.